rfcs/0064-documentation-format.md

```diff
@@ -0,0 +1,202 @@
+---
+feature: documentation-format
+start-date: 2019-12-31
+author: Silvan Mosberger (infinisil)
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: @domenkozar, @asymmetric, @alyssais, @jtojnar
+shepherd-leader: @domenkozar
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Many people from the Nix community are interested in evaluating alternatives to DocBook as the documentation format for nixpkgs/NixOS.
+Through the process of this RFC, a potentially new doc format will be decided.
+
+# Motivation
+[motivation]: #motivation
+
+The current doc format for nixpkgs (including NixOS) is DocBook, which has seen a lot of criticism over the years.
+With a more approachable and well-known documentation format we expect to have more doc contributions from more people.
+This should improve our docs to be more up-to-date and complete.
+
+# Detailed design
+[design]: #detailed-design
+
+The process for determining the doc format is as follows:
+- A set of requirements for the doc format is decided through the RFC discussion
+- Doc format candidates are collected and evaluated to see if they fulfil the requirements. This should include a small demonstration
+- A short objective overview of each valid candidate format is written, along with their advantages/disadvantages
+- A [Discourse](https://discourse.nixos.org/) post is created with these overviews, along with a poll such that people can vote on the formats they prefer. This poll will be open to the whole community and should be advertised as such
+- With the result of the poll as an input, the shepherd team decides on a doc format to be used. This decision is added to the RFC, after which it is merged.
+- People are then free to work towards that new documentation format and the committers of nixpkgs must not oppose these efforts due to the format choice
+
+## Poll
+
+Since documentation needs to be written and read by the many people from the community, we should incorporate them as possible into this important decision. Discussions in RFCs aren't optimal for this, since not many people are willing to comment, meaning the few people with the strongest opinion will be the main participants, in addition to GitHub's comment section being very annoying to navigate. Because of this, a public poll on Discourse is held instead, allowing many people to easily give their preferences.
+
+The poll is of the following form:
+- Multiple-choice, allowing people to select all formats they agree with
+- Results are only shown when the poll is closed for it to not be influenced by non-final tallies
+- Answers can be changed while the poll is still active, allowing people to discuss about formats and change their opinion (this is not optional in Discourse)
+- It runs for 1 month to give enough time for less-active people to see it
```

Comment:
I think 1 month is not enough for such a massive (and controversial change).

I propose 2 months.